rocAL_pybind/examples/rocAL_api_audio_unittest.py

```diff
@@ -0,0 +1,157 @@
+# Copyright (c) 2024 Advanced Micro Devices, Inc. All rights reserved.
```

Comment:
I think we should add a README for audio .

Wherein we add instructions to run the complete test suite for all test cases and also to induvidually run each case without the QA mode

Reply:
Done